lib/smart_answer_flows/am-i-getting-minimum-wage.rb

```diff
@@ -49,7 +51,7 @@ def define
         end
       end
 
-      use_shared_logic "minimum_wage"
+      append(Shared::MinimumWageFlow.build)
```

Comment:
In general, I think I tend to use parentheses whenever I pass any parameters to a method. Do you know why you wouldn't have used them in this case?

Is it because the call to `use_shared_logic` didn't use parentheses?

Or are you thinking that `Flow#append` is part of an internal DSL and so you want to apply [this section of the Ruby Style Guide][1]:

> Omit parentheses around parameters for methods that are part of an internal DSL (e.g. Rake, Rails, RSpec), methods that have "keyword" status in Ruby (e.g. `attr_reader`, `puts`) and attribute access methods. Use parentheses around the arguments of all other method invocations.

In any case, I don't think this is a big deal, so I'm not going to change it now.

[1]: https://github.com/bbatsov/ruby-style-guide#syntax